### ISSUE TITLE
Plume vertical displacement issue after some time.

### ISSUE THREAD

**Issue by ethanjpark** [uuvsimulator/uuv_plume_simulator#22]
Hi,
I have been running into a weird issue with the plumes where after about 2 minutes or so (150~ seconds the last time I sat and watched) a large portion of the plume is suddenly vertically displaced upwards. Sometimes this displacement happens multiple times in short succession, other times it happens once. And it happens on every run.

The particles are continuously generated at the source with the correct current velocity (which I am setting via the set_current_velocity service call).

**Comment by musamarcusso** [comment]
Hi @ethanjpark, could you please send me the parameters you used so I can try and run this here to see if it also happens on my machine?

**Comment by ethanjpark** [comment]
I managed to make this issue not happen (I hesitate to say fixed). What I suspect is that the 'current_velocity_server' instance that I created in addition to launching the gazebo world 'ocean_waves.launch' created conflicting current velocities with that produced by the gazebo current velocity controller. I took another look at the existing launch files in plume simulator and realized that the current_velocity_server was only being created if use_gazebo was false. 

Long story short I think the issue was coming from my misuse of the current_velocity_server.

In case you're still interested, here is the service call that I used for setting current velocity when I still had issues:
`rosservice call /current_velocity_server/set_current_velocity "{velocity: 1.0, horizontal_angle: 0.0}`

Thanks for the reply, the simulator's been great!
